core/src/main/java/org/apache/iceberg/TableProperties.java

```diff
@@ -129,6 +129,8 @@ private TableProperties() {
   public static final String PARQUET_BATCH_SIZE = "read.parquet.vectorization.batch-size";
   public static final int PARQUET_BATCH_SIZE_DEFAULT = 5000;
 
+  public static final String ORC_BLOOM_FILTER_COLUMNS = "orc.bloom.filter.columns";
+  public static final String ORC_BLOOM_FILTER_FPP = "orc.bloom.filter.fpp";
```

Comment:
If there's nothing that implements these properties, then I don't think that we need to include them.

Reply:
These two parameters are used for Trino, not iceberg. The relevant implementation code is in Trino.